eng/pipelines/scripts/Get-Linting-Commands.ps1

```diff
@@ -37,8 +37,7 @@ param(
   [Parameter(Mandatory = $true)]
   [string]$SourceBranch,
 
-  [Parameter(Mandatory = $true)]
-  [string]$TargetBranch,
+  [string]$TargetBranch = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "/refs/heads/"),
```

Comment:
```suggestion
  [string]$TargetBranch = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "refs/heads/"),
```

@sima-zhu recently hit issues with the leading slash so I don't think we want that.